src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -1234,7 +1234,8 @@ public void Paste()
                     newNode = NodeFactory.CreateNodeFromXml(dynEl, SaveContext.Copy);
                 }
 
-                newNode.ArgumentLacing = node.ArgumentLacing;
+                var lacing = node.ArgumentLacing.ToString();
+                newNode.UpdateValue(new UpdateValueParams("ArgumentLacing", lacing));
```

Comment:
`ArgumentLacing` cannot be set directly now, so go through backdoor API `ModelBase.UpdateValue` method.